.github/workflows/diff-change-to-dist.yaml

```
@@ -42,6 +42,7 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const fs = require('fs').promises
+            const commentOnPr = require('./shared/stats/comment.js').commentOnPr
```

If you'd prefer to write the new script in ESM it's compatible with CommonJS via `await import()`

```suggestion
            const { commentOnPr } = await import('${{ github.workspace }}/shared/stats/comment.mjs')
```

The example came from [their docs](https://github.com/actions/github-script/tree/v6/#use-esm-import) but I'm assuming `process.env.GITHUB_WORKSPACE` would work too (as we've used for reading the file a bit further down)